onnx/defs/__init__.py

```diff
@@ -25,6 +26,7 @@
 ONNX_DOMAIN = ""
 ONNX_ML_DOMAIN = "ai.onnx.ml"
 AI_ONNX_PREVIEW_TRAINING_DOMAIN = "ai.onnx.preview.training"
+AI_ONNX_PNP_DOMAIN = "ai.onnx.pnp"
```

Comment:
Just thinking aloud: I wonder if "ai.onnx.linalg" would be better?

Reply:
I would think `ai.onnx.linalg` is a clearer domain name